src/main/java/com/sonyericsson/jenkins/plugins/bfa/model/FailureReader.java

```diff
@@ -177,6 +177,10 @@ public FoundIndication scan(Run build, PrintStream buildLog) {
             int currentLine = 1;
             String line;
             while ((line = reader.readLine()) != null) {
+                if (StringUtils.isEmpty(line)) {
+                    currentLine++;
```

Comment:
currentLine++ should be moved before the isEmpty check, and the second increment after the for-loop should be deleted.
otherwise I like this change, could save some CPU time.

Reply:
I think the current implementation is actually right. If you move the `currentLine++` before the `if` and remove the one after the `for`, then you should initialize `int currentLine = 0` on line 177 instead of `1`.